docs/builtin.rst

```diff
@@ -590,42 +597,49 @@ Available
 
    * - Name
      - Keys
+     - Spaces
      - Version added
      - Description - Publication
    * - Vickery-Patil (Gray Matter)
      - | ``GM_prob0.2``
+     - ``MNI152Lin6Asym``
      - 0.0.1
      - | Vickery, Sam, & Patil, Kaustubh. (2022).
        | Chimpanzee and Human Gray Matter Masks [Data set]. Zenodo.
        | https://doi.org/10.5281/zenodo.6463123
    * - Vickery-Patil (Cortex + Basal Ganglia)
      - | ``GM_prob0.2_cortex``
+     - ``MNI152Lin6Asym``
```

Comment:
How sure are we that is the Linear 6th Gen Asymmetric?

Reply:
The mask was made with CAT12 which uses SPM12 and that uses its own template which is similar to MNI Linear 6th Generation Asymmetric, at least that's what I found out given the not-so-proper documentation.

Reply:
Can you check with @kaurao and Sam? They might have change the template. Mainly what I'm doubting is the Linear/NLinear difference.

Reply:
I checked and it's based on the TPM and Shooting GM masks from CAT12 which uses SPM12. SPM12 uses `IXI549Space` as its space and on the MNI side it's closest to `MNI152Lin6Asym`.